<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryElementView</class>
 <widget class="QWidget" name="LibraryElementView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="d_artists"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l_artists">
     <property name="text">
      <string>Artist(s)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="l_remixers">
     <property name="text">
      <string>Remixer(s)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="l_title">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="l_releaseDate">
     <property name="text">
      <string>Release Date</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="d_remixers"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="d_title"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="d_releaseDate"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="l_label">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="d_label"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="l_bpm">
     <property name="text">
      <string>BPM</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="l_key">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="d_bpm"/>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="d_key"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
